tests/examples/pacer/nda/ca2_4.json

```diff
@@ -15,7 +15,8 @@
           "pacer_case_id": "55129",
           "pacer_doc_id": "00208721516",
           "pacer_magic_num": "b775e9908ad79ce2",
-          "pacer_seq_no": null
+          "pacer_seq_no": null,
+          "short_description": "Defective Document FILED"
```

Comment:
"FILED" seems maybe wrong again.

Reply:
In this case, the description on the subject is:

![Screenshot 2023-04-21 at 16 42 58](https://user-images.githubusercontent.com/486004/233746334-497bfd37-c6fa-401f-be9f-c10c67214cd6.png)

And in the footer, it doesn't contain the word "FILED", but it contains underscores between each word.
![Screenshot 2023-04-21 at 16 43 18](https://user-images.githubusercontent.com/486004/233746356-8b141f4f-06a1-4f9f-9a6f-f4fb6a13a5d9.png)